chapters/annotations.tex

```diff
@@ -165,7 +165,9 @@ \subsubsection{Plot Curves}\doublelabel{plot-curves}
 \end{lstlisting}
 
 The mandatory \lstinline!x! and \lstinline!y! expressions are restricted to be
-component references refering to a scalar variable or \lstinline!time!.
+component references refering to a scalar variable or \lstinline!time!. If
+\lstinline!x! or \lstinline!y! does not designate a variable the behavior is
+unspecified.
```

Comment:
I think we should take the chance of defining this to either be an error or explicitly allowed with a defined result.

Reply:
Discussion continued in a suggestion.